src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -409,6 +416,34 @@ func (q *qemuAmd64) setupSEVGuestPreAttestation(ctx context.Context, config sev.
 	return attestationId, nil
 }
 
+func getCPUSig(cpuModel string) sev.VCPUSig {
+	// This is for the special case for SNP (see cpuModel()).
```

Comment:
Notable that we will support SEV-ES even on SNP machines although I forget what this cpuModel exception is about.

Reply:
Yes. When starting in SNP mode the cpuModel must be `EPYC-v4` instead of `host` (this might be fixed in a newer QEMU-for-SNP tree, if such tree will ever be posted).

Gathering all the info needed for verifying SNP attestation is a task for another PR...